view/frontend/templates/checkout/success.phtml

```diff
@@ -25,7 +25,32 @@
  * @var \Adyen\Payment\Block\Checkout\Success $block
  */
 ?>
-<?php if ($block->isBoletoPayment()): ?>
+
+
+<?php if ($block->renderAction()): ?>
+    <script type="text/javascript">
+        require([
+            'jquery',
+            'adyenCheckout3101'  // the alias for "mage/accordion"
+        ], function ($, AdyenCheckout) {
+            var action = JSON.parse('<?php echo $block->getAction(); ?>');
+            var checkoutComponent = new AdyenCheckout({
+                locale: '<?php echo $block->getLocale(); ?>',
+                environment: '<?php echo $block->getEnvironment(); ?>',
+                originKey: '<?php echo $block->getOriginKey(); ?>'
+            });
+            var actionNode = document.getElementById('ActionContainer');
+            checkoutComponent.createFromAction(action).mount(actionNode);
```

Comment:
Remove the extra actionNode, checkout component can handle it
Put the createFromAction into a try catch

```suggestion
            try {
                checkoutComponent.createFromAction(action).mount('#ActionContainer');
            } catch(err) {
                // Action component cannot be created
            }
```